yggdrasil_decision_forests/learner/distributed_gradient_boosted_trees/BUILD

```diff
@@ -9,12 +9,12 @@ package(
 # =======
 
 cc_library_ydf(
-    name = "distributed_gradient_boosted_trees",
+    name = "dgbt",
```

Comment:
The long path issue is really annoying. This will also require a change in tensorflow_decision_forests/tensorflow/BUILD and possibly parts of the documentation, but we can do this later I think

Reply:
Adding todo